guides/source/action_controller_overview.md

```diff
@@ -178,48 +206,64 @@ NOTE: Support for parsing XML parameters has been extracted into a gem named `ac
 
 ### Routing Parameters
 
-The `params` hash will always contain the `:controller` and `:action` keys, but you should use the methods [`controller_name`][] and [`action_name`][] instead to access these values. Any other parameters defined by the routing, such as `:id`, will also be available. As an example, consider a listing of clients where the list can show either active or inactive clients. We can add a route that captures the `:status` parameter in a "pretty" URL:
+Parameters specified as part of a route declaration in the `routes.rb` file are
+also made available in the `params` hash.
+
+For example, we can add a route that captures the `:status` parameter for a
+client:
 
 ```ruby
 get '/clients/:status', to: 'clients#index', foo: 'bar'
```

Comment:
[double quotes instead of single quotes.
](https://3.basecamp.com/3076981/buckets/35498807/todos/7116145100#__recording_7290149221)
```suggestion
get "/clients/:status", to: "clients#index", foo: "bar"
```